docs/Installation_Guide.md

```diff
@@ -212,13 +212,11 @@ Make sure to exit the postgres client before proceeding to the next steps:
 
 Change directory to where you can find the **manage.py** file (this is located in the top level directory for the project). If you are installing the project on the VM, the project will be located within the **/vagrant/vms** directory.
 
-To view the sql commands that will be generated from syncdb, run the command:
 
-    python manage.py sqlall app_name_here
+To generate the database tables run the following command:
 
-To generate the database tables that correspond to the Django models, run the command:
 
-    python manage.py syncdb
+    python manage.py migrate
 
 NOTE: In case, you get the following error django.db.utils.ProgrammingError: relation "auth_user" does not exist, while running the above command do the following
     python manage.py migrate auth
```

Comment:
Same here, add a new line and put it in code block, also remove the line `Now again try running the command python manage.py syncdb`